Gruntfile.js

```diff
@@ -520,7 +536,8 @@ module.exports = function (grunt) {
          'npm-install', */
         'cleanempty',
         'exec:clean-nls',
-        'usemin'
+        'usemin',
+        'githooks'
```

Comment:
@humphd This is required to create githook scripts. The formatting does run automatically on before every commit. My comment above 

> Before building, I suggest running "grunt eslint:src-fix" because now we don't have --fix option in default eslint task.

Only refers to the first run of the build. Eslint does not have auto-fixing set, so styling will fail the build and you will never create githook scripts. So instead you need to format code manually once, and then it will run properly on every commit.